expression/builtin.go

```diff
@@ -906,3 +910,25 @@ func GetBuiltinList() []string {
 	sort.Strings(res)
 	return res
 }
+
+func generateRetFlag(res Expression, evalType types.EvalType, args []Expression) {
```

Comment:
Could we remove the logic?

Reply:
I think your mean is I should closed this pull request?

Reply:
done